scripts/src/public/services/orders.ts

```diff
@@ -302,37 +326,50 @@ export async function getOrderHistory(
 	};
 }
 
-function openOrderDbToApi(order: db.Order): OpenOrder {
+function getMeta(order: db.NormalOrder | db.P2POrder, userId: string) {
```

Comment:
you can just access the contexts of the relevant userId instead of using the helper isXXX functions